app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.ingredients
+    @total_cost = @ingredients.sum(:cost)
+  end
+
+  private
+    def recipe_params
```

Comment:
Nice job making this strong params method, but I see it's not being called in this controller. Try to only write the code that is necessary for each user story to work.